README.md

```diff
@@ -34,55 +36,21 @@ aws cloudformation create-stack \
-SSH Access | ✅ |
-Periodic authorized_keys Refresh | ✅ |
-Periodic Instance Health Check | ✅ |
-git lfs | ✅ |
-Additional sudo Permissions | ✅ |
-RDP Access | | ✅
-Pipeline Signing | ✅ | ✅
-
 ## Security
 
-This repository hasn't been reviewed by security researchers so exercise caution and careful thought with what credentials you make available to your builds.
+This repository hasn't been reviewed by security researchers. Therefore, exercise caution and careful thought with what credentials you make available to your builds.
 
 Anyone with commit access to your codebase (including third-party pull-requests if you've enabled them in Buildkite) will have access to your secrets bucket files.
 
-Also keep in mind the EC2 HTTP metadata server is available from within builds, which means builds act with the same IAM permissions as the instance.
+Also, keep in mind the EC2 HTTP metadata server is available from within builds, which means builds act with the same IAM permissions as the instance.
 
 ### Limiting CloudFormation Permissions
 
-By default, CloudFormation will operate using the permissions granted to the
-identity of the credentials used to initiate a stack deployment or update.
+By default, CloudFormation will operate using the permissions granted to the identity of the credentials used to initiate a stack deployment or update.
 
-If you want to explicitly specify which actions CloudFormation can perform on
-your behalf, you can either create your stack using credentials for an IAM
-identity with limited permissions, or provide an [AWS CloudFormation service role](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-iam-servicerole.html).
+If you want to explicitly specify which actions CloudFormation can perform on your behalf, you can either create your stack using credentials for an IAM identity with limited permissions, or provide an [AWS CloudFormation service role](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-iam-servicerole.html).
 
-🧑‍🔬 [templates/service-role.yml](templates/service-role.yml) template contains an
-experimental service role and set of IAM Policies that list the IAM
-Actions necessary to create, update, and delete a CloudFormation Stack created
-with the Buildkite Elastic CI Stack template. The role created by this template
-is currently being tested, but it has not been tested enough to be depended on.
-There are likely to be missing permissions for some stack parameter
-permutations.
+🧑‍🔬 [templates/service-role.yml](templates/service-role.yml) template contains an experimental service role and set of IAM Policies that list the IAM Actions necessary to create, update, and delete a CloudFormation Stack created with the Buildkite Elastic CI Stack template. The role created by this template is currently being tested, but it has not been tested enough to be depended on. There are likely to be missing permissions for some stack parameter permutations.
```

Comment:
Another point @karensawrey , while I've been thinking about it, it looks like this section of text is effectively documented on [this page of our Buildkite Docs](https://buildkite.com/docs/agent/v3/elastic-ci-aws/cloudformation-service-role) too.

Therefore, would you be OK with removing this in a subsequent PR - i.e. perhaps keep the heading by instead, have a line of text that links through to the [CloudFormation service role](https://buildkite.com/docs/agent/v3/elastic-ci-aws/cloudformation-service-role) page?

I think we can also keep the link to this page in the **Recommended reading** section below too.